src/Diagnostics/CodeAnalysis/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -174,6 +174,26 @@
   <data name="InvalidSyntaxKindTypeArgumentDescription" xml:space="preserve">
     <value>DiagnosticAnalyzer's language-specific Register methods, such as RegisterSyntaxNodeAction, RegisterCodeBlockStartAction and RegisterCodeBlockEndAction, expect a language-specific 'SyntaxKind' type argument for it's '{0}' type parameter. Otherwise, the registered analyzer action can never be invoked during analysis.</value>
   </data>
+  <data name="StartActionWithOnlyEndActionTitle" xml:space="preserve">
+    <value>Start action has no registered non-end actions.</value>
+  </data>
+  <data name="StartActionWithNoRegisteredActionsMessage" xml:space="preserve">
+    <value>You did not register any analyzer action on '{0}'. Move all the action registrations in '{1}' that depend on this start action onto '{0}'.</value>
```

Comment:
@JohnHamby Your wording sounds a lot better, I'll update it accordingly. Probably you can also look at the description and suggest improvements, that has quite a bit of text.